src/Features/CSharp/Portable/Organizing/Organizers/MemberDeclarationsOrganizer.Comparer.cs

```diff
@@ -171,6 +171,7 @@ private static OuterOrdering GetOuterOrdering(MemberDeclarationSyntax x)
                     case SyntaxKind.EnumDeclaration:
                     case SyntaxKind.DelegateDeclaration:
                     case SyntaxKind.RecordDeclaration:
+                    case SyntaxKind.RecordStructDeclaration:
```

Comment:
:memo: not tested

Reply:
we haven't shipped this.  so no tests needed.